components/at_mention/at_mention.test.tsx

```diff
@@ -227,10 +229,10 @@ describe('components/AtMention', () => {
                 getBoundingClientRect: () => ({
                     top: 400,
                 }),
-            },
+            } as any,
         };
 
-        wrapper.instance().handleClick({preventDefault: jest.fn(), target: AtMention});
+        wrapper.instance().handleClick({preventDefault: jest.fn(), target: AtMention} as any);
```

Comment:
same here ... create as strictly typed objects as possible please.

Reply:
any idea for this type?

Reply:
well ... you are defining it yourself in `at_mention.tsx` => `React.MouseEvent<HTMLAnchorElement>`

Reply:
i have tried `React.MouseEvent<HTMLAnchorElement>` but `{preventDefault: jest.fn(), target: AtMention}` is not a valid `React.MouseEvent<HTMLAnchorElement>` a lot of property missing.

Reply:
Well, since we probably do not need a full fletched event for testing purposes we leave it like that.